### ISSUE TITLE
Problem with plugin debian: 'ai, adg, adu' if aptitude not installed

### ISSUE THREAD

**Issue by Mundball** [ohmyzsh/ohmyzsh#8527]
**Describe the bug**
alias ai not work when aptitude not installed

**To Reproduce**
 Enable plugin 'debian'
2. Run command 'ai.'
3. See error 
 su -lc 'aptitude -P install ' root
 zsh:1: command not found: aptitude

**Expected behavior**
apt-get what in apt_pref is defined

**Screenshots or recordings**
Nope only my solution.

```
diff --git a/plugins/debian/debian.plugin.zsh b/plugins/debian/debian.plugin.zsh
index 68c6df1..a05b347 100644
--- a/plugins/debian/debian.plugin.zsh
+++ b/plugins/debian/debian.plugin.zsh
@@ -79,12 +79,12 @@ else
     }
     alias ac="su -ls '$apt_pref clean' root"
     alias ad="su -lc '$apt_pref update' root"
-    alias adg="su -lc '$apt_pref update && aptitude $apt_upgr' root"
-    alias adu="su -lc '$apt_pref update && aptitude dist-upgrade' root"
+    alias adg="su -lc '$apt_pref update && $apt_pref $apt_upgr' root"
+    alias adu="su -lc '$apt_pref update && $apt_pref dist-upgrade' root"
     alias afu="su -lc '$apt-file update'"
     alias au="su -lc '$apt_pref $apt_upgr' root"
     function ai() {
-        cmd="su -lc 'aptitude -P install $@' root"
+        cmd="su -lc '$apt_pref  install $@' root"
         print "$cmd"
         eval "$cmd"
     }
-- 
2.20.1
```

**System:**
 - OS: debian 10
 - Zsh version [zsh 5.7.1 (x86_64-debian-linux-gnu)]
 - Terminal emulator [gnome-terminal 3.30.2]

**Additional context**
Nope, only many thanks to all helping people.

**Comment by Lyrex** [comment]
Maybe as an updated observation: The problem is that the `apt_pref` variable is entirely ignored if the `apt_upgr` variable isn't set either.
The condition (see [here](https://github.com/ohmyzsh/ohmyzsh/blob/master/plugins/debian/debian.plugin.zsh#L4)) requires that both are set so that `apt_pref` doesn't get overridden, which is in my opinion certainly not the expected behavior.

It should be either be possible to set both independently or it should be clearly documented that both need to be set.

**Comment by Mundball** [comment]
that can a problem too, the problem i see: aptitude is hard coded for a variable, look in the lines from my diff, that is also a solution

**Comment by mcornella** [comment]
Fixed in 7329b2fb.
